doc/api/v8.md

```diff
@@ -390,6 +390,83 @@ The API is a no-op if `--heapsnapshot-near-heap-limit` is already set from the
 command line or the API is called more than once. `limit` must be a positive
 integer. See [`--heapsnapshot-near-heap-limit`][] for more information.
 
+## `v8.takeGCProfile(options)`
```

Comment:
I don't feel like `take` is the correct word here. What about `collect`?

Reply:
Fixed, thanks!